engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/time/BasePeriodMinMaxValidator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.hv.time;
+
+import java.lang.annotation.Annotation;
+import java.time.Period;
+import javax.validation.ConstraintValidator;
+
+import org.hibernate.validator.internal.util.logging.Log;
+import org.hibernate.validator.internal.util.logging.LoggerFactory;
+
+/**
+ * Base class for PeriodMinMax validators containing comparing method for {@link Period}.
+ *
+ * @author Marko Bekhta
+ */
+public abstract class BasePeriodMinMaxValidator<T extends Annotation> implements ConstraintValidator<T, Period> {
```

Comment:
Note: the comments in this class are to be taken into account only if we keep the `Period` validators.